src/app/checklists/checklists.component.ts

```diff
@@ -53,6 +55,9 @@ export class ChecklistsComponent implements OnInit {
   showFilePicker = false;
   showFileUpload = false;
 
+  protected readonly FOREFLIGHT_EXTENSION = ForeFlightUtils.FILE_EXTENSION;
```

Comment:
Optional: I *think* you can reference other packages from the template, if you want to avoid this.

Reply:
I tried to pass only the variables that should be used instead of the whole class to control the use of its contents. I changed it in 416cdd7 if you like it better.